libnavigation-ui/src/test/java/com/mapbox/navigation/ui/route/MapRouteLineTest.kt

```diff
@@ -642,6 +675,20 @@ class MapRouteLineTest {
         assertEquals(LocationComponentConstants.SHADOW_LAYER, result)
     }
 
+    @Test
+    fun getBelowLayerWithNullLayerId_noSymbolLayers_2() {
```

Comment:
NIT What's the difference between `getBelowLayerWithNullLayerId_noSymbolLayers` and `getBelowLayerWithNullLayerId_noSymbolLayers_2`? Could we name it something different instead `2` to provide more context as to what's being tested?